pkg/action/package.go

```diff
@@ -104,6 +109,14 @@ func (p *Package) Run(path string, vals map[string]interface{}) (string, error)
 	return name, err
 }
 
+// validateName verifies if the chart name has any illegal characters or not
+func validateName(name string) error {
+	if matched, _ := regexp.MatchString("^[[:lower:][:digit:]-]+$", name); !matched {
+		return errors.New(fmt.Sprintf("Invalid chart name \"%s\". Chart names must be lower case letters and numbers with optionally dashes (-) for separator\n", name))
```

Comment:
Linking is noting... `should replace errors.New(fmt.Sprintf(...)) with fmt.Errorf(...)`. Can you please make this change?